src/hooks/useProductTypeDelete/useProductTypeDelete.tsx

```diff
@@ -0,0 +1,68 @@
+import { useProductCountQuery } from "@saleor/products/queries";
```

Comment:
I like having `useProductTypeDelete` and `usePageTypeDelete` as hooks to to split that logic from the PageType/ProductType List view but I think the files should live next to them instead of general `src/hooks` directory (as it suggests this hook might be reusable throughout the app).